### DISCUSSION TITLE
Make lazer support the "x users online" label in the web changelog

### DISCUSSION THREAD

**Discussion by neelkarma** [ppy/osu#18590] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/59462098/172151700-2cbef6fb-8bb4-46e0-8788-da5ab4a7377c.png)

It would be nice if we could see how many people are actively playing lazer in the changelog like it currently does for stable, beta and cutting edge.

**Comment by frenzibyte** [comment, marked as answer]
The label is supported in osu!(lazer) already. I believe you're requesting for the "Lazer" changelog stream to show the number of users online inline with other streams. This is a web-side issue and has already been [mentioned before](https://github.com/ppy/osu-web/issues/5675), we are only getting closer to it.

**Reply by peppy** [reply]
It's actually a combination of `osu` and `osu-server-spectator`. The former needs to send version data (already is as a hash, may be enough) and the latter needs to report counts on a regular basis.

**Reply by frenzibyte** [reply]
Thought it would still go through `osu-web` to allow sharing osu!(lazer)'s online users through APIv2, so the work on `osu` and `osu-server-spectator` are just dependencies for it. 

Should I open an issue thread for it at the `osu-server-spectator` side or is it tracked internally already?

**Reply by peppy** [reply]
https://github.com/ppy/osu-server-spectator/issues/138
